accounts/account_manager.go

```diff
@@ -147,6 +147,16 @@ func (am *Manager) Sign(addr common.Address, hash []byte) (signature []byte, err
 	return crypto.Sign(hash, unlockedKey.PrivateKey)
 }
 
+// VerifyKeyAndSign signs hash if the private key matching the given address
+// can be decrypted with the given authKey.
+func (am *Manager) VerifyKeyAndSign(addr common.Address, keyAuth string, hash []byte) (signature []byte, err error) {
```

Comment:
Please call `keyAuth` -> `passphrase` (to stay consistent with the rest of the account management code). Regarding the method name, it seems a bit strange to me as it doesn't convey what it really does and/or is designated for. Perhaps `SingleSing` or `DirectSign` or `SandboxedSing`? Not sure, neither is particularly good.